api/src/pcapi/core/fraud/factories.py

```diff
@@ -64,32 +64,32 @@ class Meta:
         model = models.UserProfilingFraudData
 
     account_email = factory.Sequence("user.email-{0}@example.com".format)
-    account_email_result = random.choice(USERPROFILING_RESULTS)
-    account_email_score = factory.Faker("pyint")
-    account_telephone_result = random.choice(USERPROFILING_RESULTS)
+    account_email_result = "success"
+    account_email_score = "0"
```

Comment:
Ça n'est pas plutôt un _int_ ?

```suggestion
    account_email_score = 0
```

Reply:
oups